pyproject.toml

```diff
@@ -33,6 +33,7 @@ psutil = "^5.9.4"
 types-psutil = "^5.9.5.4"
 types-toml = "^0.10.8"
 pytest-httpserver = "^1.0.6"
+aiohttp = "^3.8.3"
```

Comment:
`requests` isn't `async`, so I went with `aiohttp` here.

Also, the script is meant to be free-standing, so that we can scp it into prod pageservers and use it right away.
We don't have the `PageserverHttpClient`  there.

Reply:
aiohttp was installed as a package on servers. For the future I think using binary clients like curl and subprocesses is a simpler option